src/SFA.DAS.PR.Application/Common/Validators/UkprnFormatValidator.cs

```diff
@@ -13,5 +13,12 @@ public static class UkprnFormatValidator
                                .WithMessage(UkprnFormatValidationMessage)
                                .When(model => model.Ukprn.HasValue);
         }
+
+        public static IRuleBuilderOptions<T, long?> CheckNullableUkprnFormat<T>(this IRuleBuilder<T, long?> ruleBuilder) where T : INullableUkprnEntity
```

Comment:
I think this will invalidate in case of null, no? We need a test added for null value resulting in valid request.